modules/services/cache/variables.tf

```diff
@@ -0,0 +1,35 @@
+variable "customer_name" {}
+variable "environment" {}
+variable "edxapp_security_group_id" {}
+
+variable "redis_node_type" {
+  default = "cache.m4.large"
+}
+variable "redis_num_cache_nodes" {
+  default = 1
+}
+variable "redis_parameter_group_name" {
+  default = "default.redis6.x"
+}
+variable "redis_cluster_id" {
+  default = "edx-redis-cluster"
```

Comment:
The cluster IDs should use the `${environment}` name, since we're going to need to make stage and prod clusters (and it would be nice to be able to do that without naming everything..

but open to your views here 😄

Reply:
improved